src/buffer/out/textBuffer.cpp

```diff
@@ -2622,7 +2635,7 @@ HRESULT TextBuffer::Reflow(TextBuffer& oldBuffer,
     // printable char gets reset. See GH #12567
     auto newRowY = newCursor.GetPosition().y + 1;
     const auto newHeight = newBuffer.GetSize().Height();
-    const auto oldHeight = oldBuffer.GetSize().Height();
+    const auto oldHeight = oldBuffer._estimateOffsetOfLastCommittedRow() + 1;

```

Comment:
I added this, because it's a simple change, but speeds up reflow of empty buffers quite a lot. It needs +1 because `_estimateOffsetOfLastCommittedRow` returns the offset of the last row and not the height (= past the end) of the buffer.